src/infrastructure/Models/ValidationRulesReader.php

```diff
@@ -94,7 +93,7 @@ public function rules() : array
      * @throws InvalidOptionsException
      * @throws MissingOptionsException
      */
-    public function addType($type) : Type
+    private function addType($type) : Type
```

Comment:
where this is used?

Reply:
where is used method constainsMap from this class?

Reply:
fixed